server/base/src/main/java/org/apache/accumulo/server/metrics/MetricsInfoImpl.java

```diff
@@ -237,6 +254,41 @@ public synchronized void close() {
     Metrics.globalRegistry.close();
   }
 
+  /**
+   * Description of what the function does.
+   *
+   * @param patternList Description of what this variable is, i.e. comma-delimited regext patterns
+   * @return description of what this function returns, i.e. a predicate
+   */
+  public static MeterFilter getMeterFilter(String patternList) {
+    requireNonNull(patternList, "patternList must not be null");
+    Preconditions.checkArgument(!patternList.isEmpty(), "patternList must not be empty");
+
+    String[] patterns = patternList.split(",");
+    Predicate<Meter.Id> finalPredicate = null;
```

Comment:
I think if you initialize `finalPredicate` to `id -> false;` you'll be able to just `OR` together the predicates in the loop below and also you wont have to `requireNonNullElseGet` in the return statement.